packages/assets-controllers/jest.config.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+const path = require('path');
+const merge = require('deepmerge');
+const baseConfig = require('../../jest.config.packages');
+
+const displayName = path.basename(__dirname);
+const overwriteArrays = (_destinationArray, sourceArray, _options) =>
+  sourceArray;
+
+module.exports = merge(
+  baseConfig,
+  {
+    // The display name when running multiple projects
+    displayName,
+
+    // An object that configures minimum threshold enforcement for coverage results
+    coverageThreshold: {
+      global: {
+        branches: 88.86,
+        functions: 96.71,
+        lines: 96.62,
+        statements: 96.69,
+      },
+    },
+
+    // We rely on `window` to make requests
+    testEnvironment: 'jsdom',
+    testEnvironmentOptions: {},
+  },
+  { arrayMerge: overwriteArrays },
```

Comment:
Is this needed? There's no arrays in the config.

Reply:
I was trying to ensure that `testEnvironmentOptions` was empty or that if it wasn't, it had a `customExportConditions` property that was an empty array. Not sure what I was thinking here as I don't this code accomplishes that. The tests still pass when I remove this extra `arrayMerge` option. Fixed in [1b24bc7](https://github.com/MetaMask/controllers/pull/831/commits/1b24bc7fe4d12f09e6ba521383d23f723c9dc006).